.travis.yml

```diff
@@ -2,12 +2,13 @@ language: node_js
 node_js:
   - "node" # Latest node version
   - "lts/*" # Latest LTS version
+  - "12"
   - "10"
   - "8"
-  - "8.10"
 
-before_install: if [[ `npm -v` != 6* ]]; then npm i -g npm@6.4.1; fi
+before_install: if [[ `npm -v` != 6* ]]; then npm i -g npm@6.13.4; fi
```

Comment:
I think we want to remove this `if` statement so that we always have the same version of npm. As it is now, if the version of Node we are using comes with any version of npm 6, we won't get the version of npm we want to use everywhere (6.13.4).